protocol/README.md

```diff
@@ -25,147 +25,149 @@ Every TBDex message contains the following fields:
+| `payinInstrument`   | PaymentInstrument       | Y        | Specify which payment instrument to send counter currency. |
+| `payoutInstrument`   | PaymentInstrument       | Y        | Specify which payment instrument to receive base currency. |
+
+Alice -> PFI: "OK, that offering looks good. I want a Quote against that Offering, and here is how much counter currency I want to trade for your base currency. Here's the payment instrument I intend to pay you with, and here's the payment instrument I expect you to pay me in."
 
 ```json
 {
-  "id": "1234",
-  "have": "AUD",
-  "want": "Cheese",
-  "side": "want",
-  "size": 100
+  "rfqId": "1",
+  "pair": "BTC_USD",
+  "amount": 10.00,
+  "verifiablePresentation": { ... },
+  "payin_instrument": {
+      "kind": "DEBIT_CARD"
+  },
+  "payout_instrument": {
+      "kind": "BTC_ADDRESS"
+  }
 }
 ```
-I have AUD, and I want 100 Cheese.
 
-
-## Array of `Quote`
+## `Quote`
 | field            | data type   | required | description                                                   |
 | ---------------- | ----------- | -------- | ------------------------------------------------------------- |
-| `quoteId`        | string         | Y        | Identifier for this quote.                                 |
-| `requestForQuoteId`          | string         | Y        | The request this quote is responding to.                   |
-| `offerUnit`     | string            | Y        | What I'm offering.                                         |
-| `offerSize`     | int         | Y        | Amount that's being offered.                                         |
-| `costUnit`      | string         | Y        | What is being requested in exchange for this offer.                            |
-| `costSize`      | int            | Y        | Amount of `costUnit` that is being requested for this offer.                            |
-| `paymentType`   | JSON Object    | Y        | A normalized object describing the type of payment acceptable for this offer.                        |
-| `presentationDefinitionRequest`     | JSON Object    | Y        | PresentationRequest that describes that credential requirements needed to accept this offer |
-
+| `rfqId`          | string         | Y        | The identifier of the RFQ request this quote is responding to.|
+| `quoteId`          | string         | Y        | The identifier of this quote.|
+| `expiryTime`     | datetime         | Y        | When this quote expires.|
+| `totalFee`     | string         | Y        | Total fee (base + paymentInstrument specific) included in quote|
+| `amount`     | string         | Y        | Amount of base currency that the PFI is willing to send|
+| `paymentPresentationRequest`     | PresentationRequest   | Y        | PresentationRequest that describes the payment instrument needed to execute this Quote (with payment kind indicated per the RFQ) |
```

Comment:
so i think this would look a bit different in that this would smush 2 payment VC into 1 presentationRequest (payin and payout). 

this makes me think we don't actually need presentationRequest in the `Offering` object, since Alice is sending in `paymentVerifiablePresentation` against this `paymentPresentationRequest`. what do you think?